### PR TITLE
run_classifier.py improvements

### PR DESCRIPTION
**Pull request by rodgzilla**
Hi !

This PR contains multiple improvements to the `run_classifier.py` file. 

The changes are:
- removing trailing whitespaces ([PEP 8](https://www.python.org/dev/peps/pep-0008/)),
- simplifying a bit the data processing code, in particular tensor formatting,
- fixing issue #83 by adapting the value of the `num_labels` argument of `BertForSequenceClassification.from_pretrained` to the dataset being used.

### CODE REVIEW

**Review by thomwolf — COMMENTED** [review]
Sorry for the delay on that, was at NeurIPS for the week. Just one comment you could check before we merge please, ok?

File: examples/run_classifier.py [hunk 1]
```diff
@@ -260,10 +246,10 @@ def convert_examples_to_features(examples, label_list, max_seq_length, tokenizer
         input_mask = [1] * len(input_ids)
 
         # Zero-pad up to the sequence length.
-        while len(input_ids) < max_seq_length:
-            input_ids.append(0)
-            input_mask.append(0)
-            segment_ids.append(0)
+        padding = [0] * (max_seq_length - len(input_ids))
```
[the remark this answers is not in the document]
Comment (rodgzilla) [reply examples/run_classifier.py]: We don't need the test for functionality.

```python
In [1]: [0] * (-2)
Out[1]: []
``` 

I can add a commentary if you feel that this line can confuse readers.

[the remark this answers is not in the document]
Comment (thomwolf) [reply examples/run_classifier.py]: Oh you're right, Python is always so safe

**Comment by thomwolf** [comment]
Neat!

**Pull request merged**